public/app/features/plugins/components/AppRootPage.test.tsx

```diff
@@ -43,7 +44,7 @@ class RootComponent extends Component<AppRootProps> {
   }
 }
 
-function renderUnderRouter() {
+function renderUnderRouter(page: string | undefined) {
```

Comment:
you can mark this parameter as optional `function renderUnderRouter(page?: string) {`

And you won't have to modify the call in line 118 to pass an empty string.